hazelcast/src/main/java/com/hazelcast/internal/serialization/InternalSerializationService.java

```diff
@@ -130,4 +132,9 @@
 
     byte getVersion();
 
+    /**
+     * @return compact context
+     */
+    @Nonnull
+    CompactContext getCompactContext();
```

Comment:
It does not seem to be used except for the test variant of  `InternalSerializationService`.

Reply:
Removing this one as well.